g2p_seq2seq/g2p.py

```diff
@@ -80,10 +81,14 @@ def load_decode_model(self):
                                                             "vocab.grapheme"))
     self.ph_vocab = data_utils.load_vocabulary(os.path.join(self.model_dir,
                                                             "vocab.phoneme"))
-
-    self.rev_ph_vocab =\
-      data_utils.load_vocabulary(os.path.join(self.model_dir, "vocab.phoneme"),
-                                 reverse=True)
+    if(self.mode and self.mode == 'g2p'):
```

Comment:
Python does not require parenthesis and does not require checks like C. Also watch the spaces.

Reply:
Done!